TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst

```diff
@@ -716,8 +718,8 @@ Example::
 
 	logoUri = TYPO3.Neos:ImageUri {
 		asset = ${q(node).property('image')}
-		maximumWidth = 100
-		maximumHeight = 100
+		width = 100
+		height = 100
```

Comment:
no need to change this example, not sure if a cropped image is more common than a scaled one